t/local/head.t

```diff
@@ -0,0 +1,31 @@
+use warnings;
+use strict;
+use Test::More tests => 8;
+
+use lib qw( t t/local );
+use LocalServer;
+
+BEGIN {
+    use Tools;
+}
+
+BEGIN {
+    delete @ENV{ qw( IFS CDPATH ENV BASH_ENV ) };
+    use_ok( 'WWW::Mechanize' );
+}
+
```

Comment:
I don't think `use Tools` is doing much here. It looks like it does ` delete @ENV{ qw( http_proxy HTTP_PROXY PATH IFS CDPATH ENV BASH_ENV) };` and then exports `$canTMC memory_cycle_ok`, which aren't being used in this test.

Reply:
I've had a look at other tests, and there's [a comment in add_header.t for the `delete` line](https://github.com/libwww-perl/WWW-Mechanize/blob/900faaf6e64c6ab60cc4c6188c070b8f29b0776f/t/add_header.t#L9) that says _Placates taint-unsafe Cwd.pm in 5.6.1_. Actually it seems like nearly all the tests that actually run `get()` delete these environment variables. 

I agree we don't need `use Tools`.